README.md

```diff
@@ -36,38 +48,39 @@ To keep every CSS declaration readable, let's follow some rules:
 ## Source tree
 
 _main.css_: call all css behind
-_styleguide.css_: style only use to present styleguide
+_styleguide.css_: extends *main.css* to this present styleguide
 
 - 00-base:
```

Comment:
What is the purpose of writing the file tree while it's just [here](https://github.com/neomad-team/styleguide/tree/master/content/styles)?

Reply:
it's true!! and boring to maintain... I remove all the tree It's pretty clear I guess